KinomeScan-Appyter/Kinase Input Data Processing

```diff
@@ -0,0 +1,69 @@
+# Creates the dictionaries and converts them to GMT files
+
+import pandas as pd
+
+old_url = 'http://lincs.hms.harvard.edu/db/datasets/20000/results?small+molecules=HMS_ID&output_type=.csv'
+
+df = []
+
+kinome_df = pd.read_excel('http://lincs.hms.harvard.edu/wordpress/wp-content/uploads/2013/11/HMS-LINCS_KinomeScan_Datasets_2018-01-18.xlsx')
+kinome_df['sm_hms_id'] = kinome_df['sm_hms_id'].str.replace(r'\D', '') #remove HMSL before the ID
+
+# retrieve the CSV file for the small molecules, 2-183
+for x in range(0, 181):  
+    hms_id_string = str(kinome_df.iloc[x]['sm_hms_id'])
+    new_url = old_url.replace("HMS_ID", hms_id_string) # fills in HMS_ID with the correct small molecule 
+    
+    data = pd.read_csv(new_url)
+    df.append(pd.DataFrame(data))
```

Comment:
Instead of doing `df = []` (it's also confusing - 'df' stands 'DataFrame' but you use a list) you can do something like `df = pd.DataFrame()` and append rows csv files as rows of dataframe in bulk.
Which allows you to do filter out rows that you don't need and iterate over significantly less (?) rows that you need.
`df = df.loc[df['Binding Class'] <= 3]`

Reply:
Oh, yes, that's a good idea - some small molecules definitely have kinases that have a lot of kinases with binding class/affinities of 10